src/Fantomas.CLI.Benchmarks/Runners.fs

```diff
@@ -0,0 +1,35 @@
+module Fantomas.CLI.Benchmarks.Runners
+
+open BenchmarkDotNet.Attributes
+open BenchmarkDotNet.Engines
+open Fantomas.Core
+open System.IO
+
+let config = FormatConfig.FormatConfig.Default
+
+let (</>) x y = Path.Combine(x, y)
+
+[<MemoryDiagnoser>]
+[<SimpleJob(runStrategy = RunStrategy.ColdStart, targetCount = 3)>]
+type ColdStart() =
+
+    let tmpDir = __SOURCE_DIRECTORY__ </> ".." </> ".." </> "tmp"
+
+    //? Should we download these repositories automatically?
+    [<Params("FsToolkit.ErrorHandling", "FAKE", "fsharp", "Plotly.NET")>]
```

Comment:
Should we download these repositories automatically as part of the suite?

Reply:
I have something else in mind for that. This shouldn't be part of this PR I think.